Applications/DataExplorer/mainwindow.h

```diff
@@ -47,7 +49,6 @@ class MainWindow : public QMainWindow, public Ui_MainWindowClass
 
 public:
 	MainWindow(QWidget* parent = 0);
-	~MainWindow();
```

Comment:
Regarding the compilation errors: Maybe they go away, if you keep the destructor declaration here and define an empty destructor in `mainwindow.cpp`